lib/KalturaHelpers.php

```diff
@@ -313,6 +313,21 @@ public static function getAllowedPlayers() {
 		return $players;
 	}
 
+	private static function _filterOldPlayers($players) {
+		$allowedPlayers = array();
+		foreach($players as $player) {
+			if(!empty($player->html5Url)) {
+				$htmlPlayerUrl = $player->html5Url;
+				$htmlPlayerUrlParts = explode('/', $htmlPlayerUrl);
+				if(isset($htmlPlayerUrlParts[3]) && substr($htmlPlayerUrlParts[3], 1, 1) === '2') {
```

Comment:
Can't we do something simpler for the identification of V2 player? I see that there is a tag named `html5v2`.
You should also make the filtering out option configurable.
You can add a new feature to `features.php` called 'show_legacy_players', and if it is enabled, _filterOldPlayers will not be executed. (use `KalturaHelpers::isFeatureEnabled( 'show_legacy_players' )`). By default it will be set to false, so the plugin will only show V2 players.